packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -302,5 +309,16 @@ describe('Popover', () => {
         expect(actual).toHaveNoViolations();
       });
     });
+
+    it('should render items as role=menuitem and dividers as role=presentation', async () => {
+      renderPopover(baseProps);
+
+      const items = screen.getAllByRole('menuitem');
+      const dividers = screen.getAllByRole('presentation');
+      expect(items.length).toBe(2);
+      expect(dividers.length).toBe(1);
+
+      await flushMicrotasks();
```

Comment:
What does this do?

Reply:
It flushes Promise microtasks to prevent act() warnings:

https://github.com/sumup-oss/circuit-ui/blob/f78dd611fc3070cee8a9c2914a633889a92c778a/packages/circuit-ui/components/Popover/Popover.spec.tsx#L113-L122